app/src/lib/stores/app-store.ts

```diff
@@ -236,6 +234,7 @@ import {
 import { IStashEntry, StashedChangesLoadStates } from '../../models/stash-entry'
 import { RebaseFlowStep, RebaseStep } from '../../models/rebase-flow-step'
 import { arrayEquals } from '../equality'
+import * as moment from 'moment'
```

Comment:
:art: restore this `import` back to it's previous location

Reply:
775cdcc